gtsam/navigation/CombinedImuFactor.h

```diff
@@ -54,6 +48,61 @@ typedef ManifoldPreintegration PreintegrationType;
  *     Robotics: Science and Systems (RSS), 2015.
  */
 
+struct GTSAM_EXPORT PreintegrationCombinedParams : PreintegrationParams {
```

Comment:
Add doxygen /** */ block above?

Reply:
ok, added a quick description of the new struct